packages/aws-cdk-lib/aws-stepfunctions/lib/states/state.ts

```diff
@@ -678,3 +767,7 @@ function isPrefixable(x: any): x is Prefixable {
 function isNextable(x: any): x is INextable {
   return typeof(x) === 'object' && x.next;
 }
+
+export function _whichQueryLanguage(topLevelQueryLanguage?: QueryLanguage, stateLevelQueryLanguage?: QueryLanguage) {
```

Comment:
I would name this `getQueryLangauge`. `_` usage in an exported method seems weird to me and this is essentially get the state's query language.

Reply:
Oh, I'm sorry. It is missing doc comments with internal annotation. I was thinking of using this function for internal use. If export public function instead of internal only is better, then I modify to public function! Would you?